### DISCUSSION TITLE
Add redirect after validation

### DISCUSSION THREAD

**Discussion by Nastoashi** [sa-si-dev/virtual-select#189] [Q&A]
How to add redirect after validation where URL depends of selected option?
I was trying change alert to window.location... and it didnt work
Also i know how to do this from native select but how to do this with plugin?

**Comment by sa-si-dev** [comment]
Did you try with `form` `action` attribute?

**Reply by Nastoashi** [reply]
Yes, but the link opens without adding the selected value
And I' m trying to make it so that a value is added to the link 
 `action="https://test.com?"` and after selecting `https://test.com?option1`but opens `https://test.com?`

**Reply by sa-si-dev** [reply]
Please share the code you are using to initiate the plugin and HTML for the form element.
